homework/index.js

```diff
@@ -29,13 +29,54 @@
     return elem;
   }
 
+  function makeText(stringDateAndTime) {
```

Comment:
The name `makeText` is a bit vague and does not accurately describe what this function does. Perhaps `formatDate` would come closer?